### ISSUE TITLE
`.prisma/client/index` `MODULE_NOT_FOUND` error after syncing prisma schema using `pnpm`

### ISSUE THREAD

**Issue by Leniorko** [keystonejs/keystone#8711]
# Steps to Reproduce
I published the repo where I'm trying to set up keystone. You can check it here (I made a separate branch just in case, it will be deleted when the issue resolved): https://github.com/Leniorko/portfolio/tree/keystonejs-bug

1. Clone/download repo
2. Install dependencies
3. (optional) Run `pnpm postinstall` to make sure that prisma scheme is generated and keystone is synced
4. Run `pnpm keystone dev`

## Error
After steps from above, there will be the next error:
I want you to notice that there isn't any file that I wrote which explicitly imports or requires `.prisma/client/index`

```
Error: Cannot find module '.prisma/client/index'
Require stack:
- /home/leniorko/Programming/portfolio/node_modules/.pnpm/@prisma+client@4.15.0_prisma@4.15.0/node_modules/@prisma/client/index.js
- /home/leniorko/Programming/portfolio/node_modules/.pnpm/@keystone-6+core@5.3.2_@babel+core@7.22.9_@prisma+generator-helper@5.0.0/node_modules/@keystone-6/core/scripts/cli/dist/keystone-6-core-scripts-cli.cjs.dev.js
- /home/leniorko/Programming/portfolio/node_modules/.pnpm/@keystone-6+core@5.3.2_@babel+core@7.22.9_@prisma+generator-helper@5.0.0/node_modules/@keystone-6/core/scripts/dist/keystone-6-core-scripts.cjs.dev.js
- /home/leniorko/Programming/portfolio/node_modules/.pnpm/@keystone-6+core@5.3.2_@babel+core@7.22.9_@prisma+generator-helper@5.0.0/node_modules/@keystone-6/core/scripts/dist/keystone-6-core-scripts.cjs.js
- /home/leniorko/Programming/portfolio/node_modules/.pnpm/@keystone-6+core@5.3.2_@babel+core@7.22.9_@prisma+generator-helper@5.0.0/node_modules/@keystone-6/core/bin/cli.js
at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
at /home/leniorko/Programming/portfolio/node_modules/.pnpm/next@13.4.12_@babel+core@7.22.9_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/require-hook.js:113:36
at Module._load (node:internal/modules/cjs/loader:841:27)
at Module.require (node:internal/modules/cjs/loader:1061:19)
at require (node:internal/modules/cjs/helpers:103:18)
at Object.<anonymous> (/home/leniorko/Programming/portfolio/node_modules/.pnpm/@prisma+client@4.15.0_prisma@4.15.0/node_modules/@prisma/client/index.js:2:6)
at Module._compile (node:internal/modules/cjs/loader:1159:14)
at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
at Module.load (node:internal/modules/cjs/loader:1037:32)
at Module._load (node:internal/modules/cjs/loader:878:12) {
code: 'MODULE_NOT_FOUND',
requireStack: [
'/home/leniorko/Programming/portfolio/node_modules/.pnpm/@prisma+client@4.15.0_prisma@4.15.0/node_modules/@prisma/client/index.js',
'/home/leniorko/Programming/portfolio/node_modules/.pnpm/@keystone-6+core@5.3.2_@babel+core@7.22.9_@prisma+generator-helper@5.0.0/node_modules/@keystone-6/core/scripts/cli/dist/keystone-6-core-scripts-cli.cjs.dev.js',
'/home/leniorko/Programming/portfolio/node_modules/.pnpm/@keystone-6+core@5.3.2_@babel+core@7.22.9_@prisma+generator-helper@5.0.0/node_modules/@keystone-6/core/scripts/dist/keystone-6-core-scripts.cjs.dev.js',
'/home/leniorko/Programming/portfolio/node_modules/.pnpm/@keystone-6+core@5.3.2_@babel+core@7.22.9_@prisma+generator-helper@5.0.0/node_modules/@keystone-6/core/scripts/dist/keystone-6-core-scripts.cjs.js',
'/home/leniorko/Programming/portfolio/node_modules/.pnpm/@keystone-6+core@5.3.2_@babel+core@7.22.9_@prisma+generator-helper@5.0.0/node_modules/@keystone-6/core/bin/cli.js'
]
}
```
Also tried to run `pnpm keystone --fix` and got the same result:
![image](https://github.com/keystonejs/keystone/assets/26775798/a231ac6d-82d0-4d22-85ae-fad41c554403)

## Expected
Properly working server without error above.

## Additional info
Followed this guide from the start but with `pnpm` instead of `yarn`: https://keystonejs.com/docs/walkthroughs/lesson-1
Tried things that was suggested in #7625 but none of them worked
Tried to switch from `pnpm` to `yarn` and it's worked. This is definitevely the option to make it work, but it's not the solution that I'm seeking.

**Comment by ashuthe1** [comment]
Hi!
I am backend developer and would love to work on this issue. Can you assign this to me?

**Comment by marekryb** [comment]
@Leniorko See https://github.com/keystonejs/keystone/discussions/8345. 
TLDR: when using pnpm you have to define `prismaClientPath` in your client config.

**Comment by Leniorko** [comment]
> @Leniorko See #8345. TLDR: when using pnpm you have to define `prismaClientPath` in your client config.

Yep, this was the solution.
`prismaClientPath: "node_modules/.prisma/client"` and everything works.
You need, probably, to add information about that somewhere in the documentation. Thank you for your help.
